library/spdm_responder_lib/libspdm_rsp_algorithms.c

```diff
@@ -252,6 +252,8 @@ libspdm_return_t libspdm_get_response_algorithms(libspdm_context_t *spdm_context
 
     uint32_t other_params_support_priority_table[] = {
         SPDM_ALGORITHMS_OPAQUE_DATA_FORMAT_1,
+        SPDM_ALGORITHMS_OPAQUE_DATA_FORMAT_0,
```

Comment:
How do we support "FORMAT_0" ?

Reply:
`SPDM_ALGORITHMS_OPAQUE_DATA_FORMAT_0` is fine for libspdm. It just means that the Integrator needs to provide / parse all of the opaque data and they can't do key exchange.